src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -485,3 +486,43 @@ def list_unix_file_systems(self, file_path_name=None, file_system_name=None):
         custom_args["url"] = "{}mfs".format(self.request_endpoint)
         response_json = self.request_handler.perform_request("GET", custom_args, expected_code=[200])
         return response_json
+
+    def delete_migrated_data_set(self, dataset_name, wait=False, purge=False):
+        """
+        Deletes migrated data set.
+
+        Parameters
+        ----------
+        dataset_name
+            Name of the data set
+        
+        wait
+            If true, the function waits for completion of the request, otherwise the request is queued.
+
+        purge
+            If true, the function uses the PURGE=YES on ARCHDEL request, otherwise it uses the PURGE=NO.
+
+        Returns
+        -------
+        json
+            A JSON containing the result of the operation
+        """
+
+        data = {
+            "request": "hdelete",
+            "wait": json.dumps(False),
+            "purge": json.dumps(False),
+        }
+
+        if wait:
+            data["wait"] = json.dumps(True)
+        
+        if purge:
+            data["purge"]: json.dumps(True)
+
```

Comment:
Maybe just:
    data = {
            "request": "hdelete",
            "wait": json.dumps(wait),
            "purge": json.dumps(purge),
        }

Reply:
Thank you!